meta-zarhus-bsp/meta-zarhus-bsp-x86_64/wic/odroid-h4-uki.wks

```diff
@@ -0,0 +1,5 @@
+bootloader --timeout=0 --ptable gpt
```

Comment:
I read it as partition layout for UKI-enabled Zarhus image. Not necessarily odroid.